src/main/java/uk/gov/ons/census/fwmtadapter/services/ReceiptService.java

```diff
@@ -28,23 +28,19 @@ public ReceiptService(
 
   public void processReceipt(ResponseManagementEvent receiptEvent) {
     ActionCancel actionCancel = new ActionCancel();
-    actionCancel.setCaseId(getCaseId(receiptEvent));
-    actionCancel.setReason(RECEIPTED);
-    ActionInstruction actionInstruction = new ActionInstruction();
-    actionInstruction.setActionCancel(actionCancel);
 
-    rabbitTemplate.convertAndSend(outboundExchange, "", actionInstruction);
-  }
+    String a = receiptEvent.getPayload().getReceipt().getQuestionnaireId();
```

Comment:
`a` is not the most wonderful variable name. Also unused.

Reply:
debuggering code left in,  slap on my wrist